packages/website/src/components/Header.tsx

```diff
@@ -211,7 +207,6 @@ export default function Header({ transparent }: HeaderProps) {
           <FaGithub style={{ fontSize: "1.2em" }} />
           <HiddenMediaQuery query="max-width: 768px">
             <Spacer width={8} />
-            GitHub
```

Comment:
I think we can keep this and just adjust `768` to a value higher than that so it disappears before breaking the layout. Note that it includes the `isLarge` threshold as well.